app/script/util/util.coffee

```diff
@@ -91,7 +91,11 @@ z.util.load_url_buffer = (url, xhr_accessor_function) ->
     xhr = new XMLHttpRequest()
     xhr.open 'GET', url, true
     xhr.responseType = 'arraybuffer'
-    xhr.onload = -> resolve [xhr.response, xhr.getResponseHeader 'content-type']
+    xhr.onload = ->
+      if xhr.status is 200
```

Comment:
What happens if you get a not modified or similar? Don't we also have statics for these response codes?